### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by d-shibata7069**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by d-shibata7069 — COMMENTED** [review]
lll

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (h0tcl0g) [inline index.html:50]: lol

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```
Comment (h0tcl0g) [inline index.html:50]: lpl

File: index.html (hunk 1 above)
Comment (d-shibata7069) [inline index.html:50]: lol

**Review by d-shibata7069 — COMMENTED** [review]
lol

File: index.html (hunk 1 above)
Comment (d-shibata7069) [inline index.html:50]: ## lol

**Review by d-shibata7069 — COMMENTED** [review]
lolll

**Review by d-shibata7069 — COMMENTED** [review]
lol

**Pull request merged**